plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/BaseKuduConnectorSmokeTest.java

```diff
@@ -74,7 +74,7 @@ protected String getCreateTableDefaultDefinition()
     public void testShowCreateTable()
     {
         assertThat((String) computeScalar("SHOW CREATE TABLE region"))
-                .isEqualTo("CREATE TABLE kudu.tpch.region (\n" +
+                .isEqualTo("CREATE TABLE kudu." + getExpectedSchemaName() + ".region (\n" +
```

Comment:
You can use `matches()` instead of `isEqualTo()`. Thus, `getExpectedSchemaName` is no longer needed.

Reply:
@ebyhr Thanks! This is much better.